allensdk/brain_observatory/pickle_utils.py

```diff
@@ -0,0 +1,149 @@
+from typing import Union, Any
+import pickle
+import gzip
+import pathlib
+
+
+def load_and_sanitize_pickle(
```

Comment:
Same comment above about being in a more central location.

Reply:
moved to allensdk.core